server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -835,12 +835,18 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 				"Session metric already completed. Creating a new one.",
 				getLumberBaseProperties(this.documentId, this.tenantId),
 			);
+			const isEphemeralContainer: boolean =
+				this.sessionMetric?.properties.get("isEphemeralContainer") ?? false;
```

Comment:
Nit: can we use CommonProperties.isEphemeralContainer instead of the string everywhere?